crates/http-signature-directory/README.md

```diff
@@ -0,0 +1,62 @@
+# http-signature-directory
+
+![License](https://img.shields.io/crates/l/web-bot-auth.svg)
+[![crates.io](https://img.shields.io/crates/v/web-bot-auth.svg)][crates.io]
+
+[crates.io]: https://crates.io/crates/http-signature-directory
+
+A command-line tool to validate an HTTP message signatures directory per [the HTTP Message Signatures directory RFC](https://www.ietf.org/archive/id/draft-meunier-http-message-signatures-directory-00.html). 
+
+This is an opinionated validator:
+
+1. It does not support validating non-HTTPS URL scheme
+2. It does not support validating components other than `@authority`. Support for more thorough validation is welcome.
+
+## Tables of Content
+
+- [Features](#features)
+- [Usage](#usage)
+- [Security Considerations](#security-considerations)
+- [License](#license)
+
+
+## Usage
+
+```
+$ cargo install http-signature-directory
+
+$ http-signature-directory --help
+Usage: http-signature-directory <URL>
+
+Arguments:
+  <URL>  URL pointing to your JSON Web Key Set
+
+Options:
+  -h, --help     Print help
+  -V, --version  Print version
+
+# no output and exit code 0 indicates all valid
+$ http-signature-directory https://http-message-signatures-example.research.cloudflare.com/.well-known/http-message-signatures-directory
+
+# debug logging enabled
+$ RUST_LOG=debug http-signature-directory https://http-message-signatures-example.research.cloudflare.com/.well-known/http-message-signatures-directory
```

Comment:
content-type check seems to be missing, at least as a warning

Reply:
It's more that there is no line about checking the content-type ion the debug. I was under the impression everything would be printed

Reply:
no,  no - we will simply error out in that case. Everything that is an error will cause the application to terminate with a message indicating the error.